pennylane/compiler/compiler.py

```diff
@@ -114,7 +121,7 @@ def available(compiler="catalyst") -> bool:
 
     # It only refreshes the compilers names and entry points if the name
     # is not already stored. This reduces the number of re-importing
-    # ``pkg_resources`` as it can be a very slow operation on systems
+    # ``importlib_metadata`` as it can be a very slow operation on systems
```

Comment:
```suggestion
    # ``importlib.metadata`` as it can be a very slow operation on systems
```

Reply:
Fixed [156f5e2](https://github.com/PennyLaneAI/pennylane/pull/4786/commits/156f5e2acace8c65945e3307baca797e131d83bd) as multiple lines are affected by this change.